[33mWARNING [0m selenium.webdriver.common.selenium_manager:selenium_manager.py:137 The chromedriver version (137.0.7151.68) detected in PATH at C:\Users\Akshay\AppData\Local\Programs\Python\Python313\Scripts\chromedriver.exe might not be compatible with the detected chrome version (138.0.7204.101); currently, chromedriver 138.0.7204.94 is recommended for chrome 138.*, so it is advised to delete the driver in PATH and retry
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:58 Testcase est_CredKart_Login_002 is started
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:59 opening browser and landed the correct page-- https://automation.credence.in/login
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:66 Entering email--<selenium.webdriver.chrome.webdriver.WebDriver (session="ae27939cd6cee5fdd6dfedbfed4a5062")>
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:73 Entering password
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:80 Clicking the login button
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:97 verifying the user login
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:100 Useer login successfully
[32mINFO    [0m utilities.Logger:test_CredKart_User_Profile.py:111 Testcase test_CredKart_Login_002 is completed